### PR TITLE
mobile: remove duplicate files

### PR DESCRIPTION
**Pull request by jpsim**
These files are either unnecessary, identical, or nearly identical to files that are already present elsewhere in this repo now.

* `mobile/CODEOWNERS`
* `mobile/CODE_OF_CONDUCT.md`
* `mobile/DCO`
* `mobile/LICENSE`
* `mobile/NOTICE`
* `mobile/OWNERS.md` (merged into `OWNERS.md`)
* `mobile/PULL_REQUEST_TEMPLATE.md`

Signed-off-by: JP Simard <jp@jpsim.com>

### CODE REVIEW

**Review by Augustyniak — APPROVED** [review]

File: OWNERS.md [hunk 1]
```diff
@@ -122,3 +124,7 @@ matter expert reviews. Feel free to loop them in as needed.
   * v2 xDS, listeners, filter chain discovery service.
 * Michael Payne ([moderation](https://github.com/moderation)) (m@m17e.org)
   * External dependencies, Envoy's supply chain and documentation.
+* Cerek Hillen ([crockeo](https://github.com/crockeo)) (chillen@lyft.com)
+  * Python and C++ platform bindings.
+* Charles Le Borgne ([carloseltuerto](https://github.com/carloseltuerto)) (cleborgne@google.com)
```
Comment (alyssawilk) [inline OWNERS.md:129]: I think sadly Charles counts as Emeritus

**Review by alyssawilk — COMMENTED** [review]
should this wait until we flip source of truth?

File: OWNERS.md (hunk 1 above)
Comment (jpsim) [reply OWNERS.md:129]: Cerek too probably then.

Comment (jpsim) [reply OWNERS.md]: Kept Cerek in friends 😄

**Review by alyssawilk — APPROVED** [review]

File: OWNERS.md [hunk 2]
```diff
@@ -112,6 +112,9 @@ contributors to envoy-setec and relevant Slack channels from:
 * Jose Nino ([junr03](https://github.com/junr03)) (recruiting@junr03.com)
 * Dhi Aurrahman ([dio](https://github.com/dio)) (dio@rockybars.com)
 * Dmitry Rozhkov ([rojkov](https://github.com/rojkov)) (dmitry.rozhkov@intel.com)
+* Michael Rebello ([rebello95](https://github.com/rebello95)) (mrebello@lyft.com)
+* Alan Chiu ([buildbreaker](https://github.com/buildbreaker)) (achiu@lyft.com)
```
Comment (keith) [inline OWNERS.md:116]: if nothing else these emails are invalid now

**Review by keith — APPROVED** [review]

File: OWNERS.md (hunk 2 above)
Comment (jpsim) [reply OWNERS.md:116]: Yeah, not sure if they want their personal emails published here. Will leave as-is.

**Pull request merged**